### DISCUSSION TITLE
Change item data (name, lore, tags, ect.) after being built

### DISCUSSION THREAD

**Discussion by AstroSquared** [Minestom/Minestom#1782] [Help] [answered]
Is there a way to change item data, such as the display name, the lore, and tags after calling `ItemStack.Builder#build`, without making a new `ItemStack.Builder`?

**Comment by Tofaa2** [comment, marked as answer]
The itemstack class is immutable, you can create one and then create another one with the same features + your additions on top of it

**Reply by Tofaa2** [reply]
ItemStack.of(Material.DIRT).withName(Component.text("Hello World"), NamedTextColor.RED);

**Reply by AstroSquared** [reply]
Okay, thanks for your anwser
